src/frontend/src/pages/ProjectPage/pages/Columns.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
```

Comment:
How about scoping Columns.tsx and ContrancOverviewPage.tsx to a sub-folder?

Reply:
Hmm. Could move the entire contents of the pages folder to "ContractsOverviewPage"